UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -65,6 +65,7 @@ public class CharacterCustomization : MonoBehaviour
 
 		[SerializeField] private List<Color> availableSkinColors;
 		private CharacterSettings currentCharacter;
+		public CharacterSettings CurrentCharacter { get { return currentCharacter; } private set { new CharacterSettings(); } }
```

Comment:
wouldn't this reset the character settings every time you try to set it? If you have a public method to set it, just remove the set part.

Reply:
why would I want to set it? doesn't `private set` make sure that nobody can set anything from this variable?

Reply:
nvm fixed it.